### ISSUE TITLE
 Why does the loss always increase instead of decrease in the first stage?

### ISSUE THREAD

**Issue by Josh1021** [bassyess/CFCD#5]
I trained the network using my own dataset, but the desc_loss began to rise after the second epoch during the first stage, and the loss in the second stage behaved similarly. The parameter configuration was not altered, using R-50-1x64d_dds_8gpu.pyth as the pre-trained model.  Here is my train logs:
![1709692123208](https://github.com/bassyess/CFCD/assets/58059392/7bf1abdc-52fe-4b3f-bdc3-16488421617b)
